client/my-sites/email/email-management/home/email-plan-warnings.jsx

```diff
@@ -0,0 +1,102 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '@automattic/components';
+import Gridicon from 'calypso/components/gridicon';
+import { isEnabled } from '@automattic/calypso-config';
+import {
+	emailManagementManageTitanAccount,
+	emailManagementTitanControlPanelRedirect,
+} from 'calypso/my-sites/email/paths';
+import { TITAN_CONTROL_PANEL_CONTEXT_CREATE_EMAIL } from 'calypso/lib/titan/constants';
+import { connect } from 'react-redux';
```

Comment:
Another external dependency.

Reply:
I guess I shouldn't led the IDE do the imports :)